packages/suite/src/components/suite/notifications/NotificationRenderer/NotificationRenderer.tsx

```diff
@@ -223,7 +223,7 @@ export const NotificationRenderer = ({
                     variant="info"
                     message="EVENT_DEVICE_CONNECT"
                     messageValues={{
-                        label: notification.device.label,
+                        label: notification.device.features?.label || notification.device.name,
```

Comment:
I can see that `useSelector(selectDeviceLabelOrName)` because it'd have to be defined at top of component, above `switch`. Why not then extract this as a component, so it can call the hook internally?

But it's a nit, I guess this is fine.

Reply:
🤔 I think there's no perfect solution. e6ecafe58388ed7a484741dbf0258774b65dae10